### DISCUSSION TITLE
Proof type is not visible!

### DISCUSSION THREAD

**Discussion by smshrokib** [community/community#65142] [GitHub Education]
### Hi everyone,

The issue I’m facing is that, after uploading my proof document I cannot see the option (option is not visible in that webpage) to select the type of proof that I’m providing. And also by avoiding that I tried to process my application but it’s not getting submitted. I’m stuck in this situation for several days. I tried many many times with multiple browsers and multiple computers to solve this issue. Got no solution yet. 

If anybody can help, please !!! 

![screencapture](https://github.com/community/community/assets/130635822/70ecd4f0-a883-44db-93ac-793fde228f2f)

**Comment by arafajs** [comment]
1. Try clearing your browser's cache and cookies. This can sometimes fix problems with websites not loading properly.
2. Try using a different browser. Sometimes different browsers have different bugs, so it's worth trying a different one.
3. Try using a private browsing window. This will prevent your browser from storing any cookies or cache, which can sometimes cause problems.
4. Contact GitHub support. If you've tried all of the above and you're still having problems, you can contact GitHub support for help.

**Reply by smshrokib** [reply]
maybe I have to contact github support

**Reply by Jas0nG** [reply]
I have the same issue and all these methods above won't work, Github Support for Edu verify seems already closed, sad.

**Comment by nqlp** [comment]
You don't have to select the type of proof (because I didn't provide any). What worked for me is taking a photo with my phone and uploading it. The most important thing is the **date** because it indicates your current school enrollment status.

**Reply by smshrokib** [reply]
I tried to process my application but it’s not getting submitted.

**Comment by Jas0nG** [comment]
I have the same issue, I tried different browsers respectively and different actions, I think it's a temporary systemic issue, not sure if this only occurs when re-verifying.  
Till now, I've tried these browsers:  
- On Windows
  - Edge
  - Chrome
 - On Ubuntu
   - Chrome
   - FireFox
  - On IOS
    - safari
 And I also tried clear cache and cookies, won't work.

**Reply by ammedd** [reply]
Had the same issue trying to upload a file. Made a print and took a picture of it and that worked...

**Comment by fsantamaria1** [comment]
In my case, I uploaded a picture of an email I got from my professor congratulating me for passing my class as the proof. I think that as long as they can see that it was sent to your .edu email and is not a really old email, they will approve it. The email I uploaded was 4 months old, so I don't think it has to be extremely recent either.

So try just taking a screenshot of an email you got form someone in your school. That might help.

**Reply by Jas0nG** [reply]
Thanks a lot, even uploaded a screenshot of email doesn't work for me, I printed both of the screen shot and student card and took a pic.  
Then use PC to upload this photo and it's finally work...  So the invisible problem is still not solved, I just avoid it with more proof.

Hope this would help.

**Reply by fsantamaria1** [reply]
I hope it works.

Good luck!

**Comment by sw00z** [comment]
Have you had any luck resolving this yet? I'm still having trouble with it

**Reply by smshrokib** [reply]
Document got submitted but they didn't provide me any prompt or notification to confirm the submission. Few days ago I checked my account I saw my account was verified.

**Comment by ccj9874** [comment]
I am unfortunately experiencing this same issue. I have tried several browsers and devices, as well as both taking a new picture and uploading one, but no luck.

**Comment by Momokozo** [comment]
Hey guys! Try this! CHANGE other proof documents, TRY student ID card! I haven't found the "proof type" form, and I tried to submit my verification report of student status, but I encountered the prompt asking me to select "other" proof type. I changed another proof document and luckily succeeded!

**Reply by iHyper445** [reply]
I'm not sure this will work for this particular situation because the "Proof Type" box does not appear at all, so we can't select anything. Thank you for the tip, though!

**Reply by Momokozo** [reply]
I had the same problem as you! But I submitted successfully without selecting anything! I also met the situation the "proof type" box didn't appear. Hope it work! I guess it's because "proof type" has a default value, and exactly the student ID card meets the default type, then both params (photo and default proof type) successfully pass to the backend.
